packages/Avalonia/AvaloniaRules.Project.xml

```diff
@@ -0,0 +1,81 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Rule Name="Avalonia" DisplayName="Avalonia" Order="70" PageTemplate="generic" xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <Rule.Categories>
+    <Category Name="Compiling" DisplayName="Compiling" />
+    <Category Name="Debug" DisplayName="Debugging" />
+    <Category Name="NameGenerator" DisplayName="Nameming Generator" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile" HasConfigurationCondition="false" Label="Avalonia" />
+  </Rule.DataSource>
+
+  <!-- Compiling -->
+  <BoolProperty Name="AvaloniaUseExternalMSBuild"
+                DisplayName="External MSBuild"
+                Description="Use dotnet build instead of MSBuild"
+                Category="Compiling" />
+
+  <BoolProperty Name="AvaloniaUseCompiledBindingsByDefault"
+              DisplayName="Use CompiledBindings"
+              Description="Use by default CompiledBindings"
+              Category="Compiling" />
+
+  <BoolProperty Name="AvaloniaAccessUnstablePrivateApis"
+            DisplayName="Use Private Apis"
+            Description="Enable access to unstable private APIs (not recommended)"
+            Category="Compiling" />
+
+  <!-- Debug -->
+
+  <BoolProperty Name="AvaloniaXamlIlDebuggerLaunch"
+          DisplayName="Debug Xaml Compaliler"
+          Description="Allow debug Xaml compilation."
+          Category="Debug" />
+
+
+  <EnumProperty Name="AvaloniaXamlReportImportance"
+                DisplayName="Xaml Report Importance"
```

Comment:
Everything "Xaml" is shown to the user is incorrect following Microsoft's capitalization rules. "Xaml" is correct in C# code but "XAML" is what should be used in documentation.

https://learn.microsoft.com/en-us/dotnet/desktop/wpf/xaml/?view=netdesktop-7.0